kool.yml

```diff
@@ -22,4 +22,7 @@ scripts:
   # Generate documentation for kool commands
   make-docs:
     - rm -f docs/4-Commands/*.md
-    - kool docker golang:1.15.0 go run docs.go
+    - kool docker --volume=kool_gopath:/go golang:1.15.0 go run docs.go
+  parse-presets:
```

Comment:
did this simply override the `parse-presets` above? maybe we should remove that one hehe

Reply:
![image](https://user-images.githubusercontent.com/1116377/96630698-c19f7700-12eb-11eb-930e-57f6d5a4c444.png)

Reply:
ooops